build-tools/manifest-attribute-codegen/manifest-definition.xml

```diff
@@ -96,6 +96,10 @@
         <a name='knownActivityEmbeddingCerts' api-level='33' />
         <a name='enableOnBackInvokedCallback' format='boolean' api-level='33' />
         <a name='allowCrossUidActivitySwitchFromBelow' format='boolean' api-level='35' />
+        <a name='pageSizeCompat' api-level='36' />
+        <a name='intentMatchingFlags' api-level='36' />
```

Comment:
Should these be exposed as properties on `ApplicationAttribute`?  That's not in this PR, though `PageSizeCompat` is here.  How do we decide which ones should be exposed on attributes?

Reply:
> How do we decide which ones should be exposed on attributes?

It is definitely arbitrary.  None of them are [documented](https://developer.android.com/guide/topics/manifest/application-element) (yet?), so it's hard to know what's intended for common use and what is intended to be "undocumented".  In general, we tend to wait for people to request them.

`pageSizeCompat` is at least documented in the [behavior changes documentation](https://developer.android.com/about/versions/16/behavior-changes-all#16-kb-compatibility-mode) and seemed like it might be useful if 16KB support actually becomes an issue, so I added it.